lib/pages/new_private_chat/widget/contact_status_widget.dart

```diff
@@ -26,21 +25,14 @@ class ContactStatusWidget extends StatelessWidget {
           SvgPicture.asset(
             ImagePaths.icStatus,
             // ignore: deprecated_member_use
-            color: status == ContactStatus.active ? activeColor : inactiveColor,
+            color: inactiveColor,
```

Comment:
Pls use `colorFilter: ColorFilter.mode()` and remove ignore

Reply:
done